src/parse_relative_time.rs

```diff
@@ -278,6 +284,15 @@ mod tests {
         Ok(parsed - now)
     }
 
+    #[test]
+    fn test_empty_string() {
+        let now = Utc::now();
+        assert_eq!(
+            parse_relative_time_at_date(now, "").unwrap(),
+            now.checked_add_months(Months::new(0)).unwrap()
```

Comment:
Calling `checked_add_months` seems unnecessary and you can simplify it to just:
```suggestion
            now
```

Reply:
I think without this there was a type mismatch related to chrono's `DateTime<Local>` vs `DateTime<FixedOffset>` or something like that. I'll see if there's another way around this.

Reply:
(The same pattern appears in a few other places below, as well.)

Reply:
Nevermind, I was misremembering. Your suggestion works, I'll update it.